fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -1404,6 +1404,9 @@ struct MetaclusterManagementWorkload : TestWorkload {
 			} else if (e.code() == error_code_invalid_metacluster_operation) {
 				ASSERT(!self->metaclusterCreated);
 				return Void();
+			} else if (e.code() == error_code_tenant_locked) {
+			   ASSERT(exists);
```

Comment:
I think this can be stronger -- something like ASSERT(exists && locked && id != newID). Also, we could ASSERT the opposite in the success case when we lock a tenant.

Reply:
I'm also not really sure why `invalid_tenant_configuration` is included in this list. I suspect I might have added it by accident since I don't think there's a way that we should hit this error.

Reply:
Let me try to remove it.

Also, by `locked` in the assertion, are you referring to the `tenantState != UNLOCKED`?

Reply:
The condition for getting the tenant locked error is that the tenant is already locked and you specified a non-matching lock ID.

Reply:
Done